ground/src/test/java/com/google/android/ground/repository/SurveyRepositoryTest.java

```diff
@@ -70,11 +69,10 @@ public void testActivateSurvey_managersCanAddFeaturesToAllJobs() {
 
     surveyRepository.activateSurvey("id");
 
-    Job expectedJob = job.toBuilder().setUserCanAdd(LocationOfInterestType.ALL).build();
     surveyRepository
         .getActiveSurvey()
         .test()
-        .assertValue(p -> p.get().getJobs().equals(ImmutableList.of(expectedJob)));
+        .assertValue(p -> p.get().getJobs().equals(ImmutableList.of(job)));
   }
```

Comment:
Do we even need these test anymore? Earlier it was asserting that the inserted job was modified based on the user role. Now we are checking that the job is unmodified.

Reply:
Good catch! Removed!